### ISSUE TITLE
Welcome message read-only

### ISSUE THREAD

**Issue by LandKingdom** [portacle/portacle#100]
Hello!
I was trying Portacle for Windows out since I wanted to start learning Lisp (CL... Other dialects later maybe) and it has been recommended for me since I was having trouble setting Emacs up from scratch (MELPA and Sly). 

After the first few runs (setup and switch so Sly) I tried following the suggestion presented in the welcome message, namely, using that message as a scratchpad to write stuff. Turns out it's read only and I can't write anywhere BUT the first even char of the buffer. I thought you might be interested in this and that it could case scratches to others down the line, so I went ahead and filed this issue. Cya!

TL;DR
\*scratch\*'s startup text is read only from the 2nd char to the end of the buffer, making it writable only before the welcome text
